src/prelude/internals.mo

```diff
@@ -387,13 +387,15 @@ module @ManagementCanister = {
 let @ic00 = actor "aaaaa-aa" :
   actor {
     create_canister : {
-      settings : ?@ManagementCanister.canister_settings
+      settings : ?@ManagementCanister.canister_settings;
+      sender_canister_version : ?Nat64
```

Comment:
This is like a new (optional) keyword argument to the call, the only thing a user might reasonably want is suppressing that the controller's version gets passed automatically...